doc/da65.sgml

```diff
@@ -542,6 +542,18 @@ code. The following attributes are recognized:
   range, where <tt/label/ is the label name given with the <tt/NAME/
   attribute, and <tt/offs/ is the offset within the data.
 
+  <tag><tt>PARAMSIZE</tt></tag>
+  This optional attribute is followed by a numerical value. It tells the
+  assembler that subroutine calls to this label follow "inline parameters"
+  of the given bytes like this:
```

Comment:
That sentence needs to be reworded to say, '... subroutine calls to this label are followed by "inline parameters" with the given number of bytes, like this:'.